src/rop32.c

```diff
@@ -241,6 +247,10 @@ rop32_init(char *fp, char *boldfp, int h, int *fn_width, int *fn_height,
 	}
 skipboldscaler:
 
+	if (self->dopivot) {
+		FT_Set_Transform(self->face, &rotate_right, NULL);
```

Comment:
You should also call FT_Set_Transform for self->boldface, if we have a bold font defined.
i.e. something like:
  if (self->boldfid.file_path != NULL)
          FT_Set_Transform(self->boldface, &rotate_right, NULL);